requery/src/main/java/io/requery/sql/platform/Generic.java

```diff
@@ -19,17 +19,9 @@
 import io.requery.query.Expression;
 import io.requery.query.element.LimitedElement;
 import io.requery.query.element.OrderByElement;
-import io.requery.sql.GeneratedColumnDefinition;
-import io.requery.sql.IdentityColumnDefinition;
-import io.requery.sql.Mapping;
-import io.requery.sql.Platform;
-import io.requery.sql.UserVersionColumnDefinition;
-import io.requery.sql.VersionColumnDefinition;
-import io.requery.sql.gen.Generator;
-import io.requery.sql.gen.LimitGenerator;
-import io.requery.sql.gen.OffsetFetchGenerator;
-import io.requery.sql.gen.OrderByGenerator;
-import io.requery.sql.gen.UpsertMergeGenerator;
+import io.requery.query.element.QueryElement;
+import io.requery.sql.*;
+import io.requery.sql.gen.*;
```

Comment:
Please avoid changing the imports to wildcard, (avoid wildcard imports)

Reply:
Sorry, It was by IntelliJ Settings, reformat at home